starter-code/server.js

```diff
@@ -55,6 +58,7 @@ app.get('/articles', function(request, response) {
 app.post('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
   // Put your response here...
+  // In this line server sends the information it got from the client to the database and inserts it into a raw. (It's called from Article.prototype.insertRecord()) It's steps 2,3,4,5 and "update" part of CRUD
```

Comment:
So instead of update, this would be create. Because with app.post() you usually are creating a resource.